tools/xcms_refine/abims_xcms_refine.xml

```diff
@@ -0,0 +1,265 @@
+<tool id="abims_xcms_refine" name="xcms refineChromPeaks (refine)" version="3.12.0+galaxy1">
```

Comment:
Since it's the first version of the tools, I'd use `3.12.0+galaxy0`

Reply:
Very good.

Reply:
Done